_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,198 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope), except if the element has an implicit semantic role that is identical to its explicit semantic role.
```

Comment:
Rather than saying "explicit semantic role", should we say "has a role attribute which sets the semantic role to", or something like it. It's a bit more verbose, but we don't have a definition for "explicit". I don't even know that that's what it is called in any of the specs.

Reply:
The term "explicit role" is used generously throughout the WAI-ARIA spec.

Our own definition of "semantic role", contains this wording: 

> "[Roles](https://www.w3.org/TR/wai-aria-1.1/#dfn-role) can be [implicit](https://www.w3.org/TR/wai-aria/#implicit_semantics) through the element type or explicit through the [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute)."

We are also already using the term "explicit semantic role" in this published rule: 
https://auto-wcag.github.io/auto-wcag/rules/SC4-1-2-role-has-required-states-and-properties.html

I say we keep the definition. 
But do we need to add these two definitions, to be able to talk about implicit and explicit semantic roles? 

- Explicit semantic role: [Semantic roles](#semantic-role) can be explicit through the [role attribute](https://www.w3.org/TR/html/dom.html#aria-role-attribute).
- Implicit semantic role: [Semantic roles](#semantic-role) can be [implicit](https://www.w3.org/TR/wai-aria/#implicit_semantics) through the element type.

Reply:
@WilcoFiers, can I consider your comment resolved?